analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ExecutingSqlQueries_Net46.vb

```diff
@@ -109,7 +96,7 @@ Namespace Tests.Diagnostics
             command.CommandText = "SELECT * FROM mytable WHERE mycol=" & param ' Noncompliant
             command.CommandText = $"SELECT * FROM mytable WHERE mycol={param}" ' Noncompliant
             Dim adapter = New OdbcDataAdapter(String.Format("INSERT INTO Users (name) VALUES (""{0}"")", param), "") ' Noncompliant
-            Dim adapter1 = New odbcdataadapter(sTRing.foRmAt("INSERT INTO Users (name) VALUES (""{0}"")", param), "") ' Noncompliant
+            Dim adapter1 = New odbcdataadapter(String.Format("INSERT INTO Users (name) VALUES (""{0}"")", param), "") ' Noncompliant
```

Comment:
This is little hard to maintain, but the CraZY caSINg should be preserved to demonstrace case insensivity for VB.NET